packages/babel-preset-env/src/normalize-options.js

```diff
@@ -179,6 +183,15 @@ export default function normalizeOptions(opts: Options) {
 
   checkDuplicateIncludeExcludes(include, exclude);
 
+  let polyfills = validateInjectPolyfillsOption(opts.polyfills);
+  let useBuiltIns;
+  if (typeof opts.useBuiltIns === "string") {
+    console.warn("'useBuiltIns' has been renamed to 'polyfills'.");
+    polyfills = validateInjectPolyfillsOption(opts.useBuiltIns);
+  } else {
+    useBuiltIns = opts.useBuiltIns;
```

Comment:
whats the use case for passing `useBuiltIns` further down? Shouldnt it just get converted here to `polyfills`?

Reply:
Pardon me? `useBulitIns` is a new option (has a different functionality) now?

Reply:
My bad. I was under the impression that this PR was only about changing `useBuiltIns` to `polyfills`. Disregard my comment, you are right 👍